### ISSUE TITLE
`prefer-export-from`: false positive with conversion of default to named export

### ISSUE THREAD

**Issue by silverwind** [sindresorhus/eslint-plugin-unicorn#2161]
```js
import foo from 'module';
export {foo};
```

The rule incorrectly flags this to be rewritten as `export ... from`, but I can not do that because the intention is to convert the default export to a named export.

**Comment by fisker** [comment]
Shouldn't this auto-fixable? If not, it's a bug..

This should be prefered.

```
import {default as foo} from 'module';
```

**Comment by silverwind** [comment]
Actually you have a good point, it didn't occur to me that using the `default` export would be viable. The module in question is CommonJS, but I guess `default` will work on those as well.

**Comment by fisker** [comment]
I just checked that we have this case tested. https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/test/snapshots/prefer-export-from.mjs.md#invalid-28 So you must have other code makes it not auto-fixed.

**Comment by silverwind** [comment]
Trying the autofix now...

**Comment by silverwind** [comment]
Yes, it did it almost perfectly, the only complaint I have it the position of the new export at the end of the file, but I guess the plugin can't possibly know where I want my exports 😆.

**Comment by fisker** [comment]
Maybe more reasonable to put it at the position when you write your `export {foo};`?

**Comment by silverwind** [comment]
Yes, that would be ideal. If the `export` in question is not to be removed because of other named exports, insert the new export on the next line after it.
